### ISSUE TITLE
Failed to import Expense report-1.0.proc

### ISSUE THREAD

**Issue by vlim** [Bonitasoft-Community/expense-report-example#1]
Hi Philippe,

I'm using Bonita BPM Community Edition
Version: 7.0.3
Build id: 7.0.3

I tried to import this diagram and got the following error message:

Failed to import Expense report-1.0.proc

Please let me know what I can do to get around this.

I also tried to follow the tutorial videos that you made, related to this Expense Report example, and I came across an issue related to showing the name of the requester in an Input widget.  Please see the attached screenshots.  The Input widget looks fine in the Page Editor but does not seem to work in Preview mode and during runtime.

Maybe I missed some things.  

I would appreciate it if you could give me some pointers on where to look.

Thanks,
-Victor

![bonita_page_editor_input_with_createdby](https://cloud.githubusercontent.com/assets/2251204/10408659/dc670882-6eb8-11e5-9d07-5bb3edcbb15d.png)

![bonita_preview_input_with_createdby](https://cloud.githubusercontent.com/assets/2251204/10408663/e638aff0-6eb8-11e5-9931-46d767ff9228.png)

**Comment by pozil** [comment]
Hi,

Thanks for reporting this issue. I can reproduce it.
This was caused by a change in Bonita BPM 7.0.3.
I will release a new version for 7.0.3 and onward as soon as possible.

Cheers

**Comment by pozil** [comment]
In the meantime: the reporter name can be set by using an variable of type `Expression` and this value:

``` javascript
if (angular.isUndefined($data.createdByUser))
    return "";
return $data.createdByUser.firstname +' '+ $data.createdByUser.lastname;
```

**Comment by pozil** [comment]
Issue fixed, please use [release v2.0](https://github.com/Bonitasoft-Community/expense-report-example/releases/tag/2.0).

**Comment by vlim** [comment]
Thanks, Philippe!

-Victor
